doc/content/examples/divertor_monoblock/index.md

```diff
@@ -0,0 +1,257 @@
+respectively and simulate the trapping/release behavior of hydrogen isotopes in/from trap sites.
+For the conservation of energy equation, `SpecificHeatConductionTimeDerivative` and `HeatConduction` solve the 1st and 3rd terms of [eq:temperature_weak].
+`ADTimeDerivative,` `ADMatDiffusion,` `SpecificHeatConductionTimeDerivative,` and `HeatConduction` are MOOSE kernels commonly
+used to solve the time derivative, material diffusion in the material, and heat conduction equations in the material.
+
+!listing test/tests/divertor_monoblock/divertor_monoblock.i link=false block=Kernels
+
+Nodal kernels are used for non-diffusive variables (trapped species).
+
+!listing test/tests/divertor_monoblock/divertor_monoblock.i link=false block=NodalKernels
+
+
+### Boundary conditions and history
+
+For the mass conservation equation, we use `FunctionNeumannBC` and `DirichletBC` to solve the 2$^{nd}$ term of [eq:concentration_weak].
+`FunctionNeumannBC` is used to treat the time-dependent plasma exposure at the top (plasma-exposed) surface,
+and `DirichletBC` is used to set the BC of the solute T atom concentration to zero at the inner CuCrZr tube (at $r = 6$ mm).
+For the energy conservation equation, we use `FunctionNeumannBC` and `DirichletBC` to solve the 2$^{nd}$ term of [eq:temperature_weak].
+`FunctionNeumannBC` is used to treat the time-dependent heat flux at the top (plasma-exposed) surface,
+and `DirichletBC` is used to treat the temperature increase in the cooling tube.
+`FunctionNeumannBC,` `FunctionDirichletBC,` and `DirichletBC` are MOOSE BCs commonly used to determine the BCs of the variables to be solved.
+
+We simulate a 1,600 s-cycle plasma discharge, with each cycle consisting of a 100 s plasma ramp-up,
+a 400 s steady-state plasma discharge, a 100 s plasma ramp-down, and a 1,000 s waiting phase.
+Up to 50 cycles are simulated to achieve the total 20,000 s steady-state plasma discharge.
+[fig:tritium_temperature_history] shows the integrated (solute, total and trapped) tritium concentration in the monoblock.
+It shows that implantation fluxes and temperatures varies linearly up to (from) their steady-state values from (up to) their initial values during ramp-up (ramp-down).
+During the steady-state plasma discharge, we set a heat flux of 10 MW/m$^{2}$ at the top of the 2D monoblock (at $y = 14.0$ mm)
+and a cooling temperature of 552 K at the inner CuCrZr tube (at $r = 6.0$ mm).
+We assume 100% T plasma with a 5.0 $\times$ 10$^{23}$ m$^{-2}$$\cdot$s$^{-1}$ plasma particle flux
+(which is 50% of 1.0 $\times$ 10$^{24}$ DT m$^{-2}$$\cdot$s$^{-1}$ plasma particle flux)
+and only 0.1% of the incident plasma particle flux (5.0 $\times$ 10$^{20}$ m$^{-2}$$\cdot$s$^{-1}$) entered the first layer of mesh at $y = 14.0$ mm.
+We treated this plasma exposure by setting the flux BC (5.0 $\times$ 10$^{16}$ m$^{-3}$$\cdot$s$^{-1}$) of the solute T atom concentration in the first layer of mesh
```

Comment:
Can you explain how you get this flux and unit? Also, the way this is presented in somewhat dense and hard to follow given the back-to-back flux quantities and scientific notation. I recommend an adjustment for readability.

Reply:
Hopefully the updated documentation clarifies things.